ui/src/components/App.tsx

```diff
@@ -70,20 +70,5 @@ const App: React.FC = () => {
   )
 }
 // APP_END
-
-const PublicProvider: React.FC = ({ children }) => {
```

Comment:
This wasn't being used anymore since @alexmatson-da changed to the custom query stream provider